cmip6_downscaling/data/cmip.py

```diff
@@ -230,21 +231,27 @@ def get_gcm(
         chunking_approach=chunking_approach,
         output_path=rechunked_path,
     )
+    print('ds_gcm_rechunked:')
+    print(ds_gcm_rechunked.chunks)
 
     return ds_gcm_rechunked
 
 
 def get_gcm_grid_spec(
     gcm_name: Optional[str] = None, gcm_ds: Optional[Union[xr.Dataset, xr.DataArray]] = None
 ) -> str:
-    if gcm_ds is None:
-        assert gcm_name is not None, 'one of gcm_ds or gcm_name has to be not empty'
-        gcm_grid = load_cmip(
-            source_ids=gcm_name,
-            return_type='xr',
-        ).isel(time=0)
-    else:
-        gcm_grid = gcm_ds.isel(time=0)
+    with dask.config.set(**{'array.slicing.split_large_chunks': False}):
+
+        if gcm_ds is None:
+            """Silences the /srv/conda/envs/notebook/lib/python3.9/site-packages/xarray/core/indexing.py:1228: PerformanceWarning: Slicing is producing a large chunk. To accept the large
+            chunk and silence this warning, set the option >>> with dask.config.set(**{'array.slicing.split_large_chunks': False}):"""
```

Comment:
multi-line strings are executed at run time and since this is just a comment, the best practice is to just write as a comment.

```suggestion
            # Silences the /srv/conda/envs/notebook/lib/python3.9/site-packages/xarray/core/indexing.py:1228: PerformanceWarning: Slicing is producing a large chunk. To accept the large
            # chunk and silence this warning, set the option >>> with dask.config.set(**{'array.slicing.split_large_chunks': False}):"""
```